### PR TITLE
E2E test tutorial - waitFor Statements

### LINKED ISSUE #360: Create Tutorial - How to fix a CI flake – E2E Tests - waitFor statements
## Skills covered

Debugging,
Understanding Error Logs,
Debugging Docs, Running e2e test in local server.

## Example

Run a git patch file to reproduce 

waitFor statements in E2E tests–https://github.com/oppia/oppia/issues/16102

### PR DESCRIPTION
**Pull request by Ash-2k3**
Fix https://github.com/oppia/oppia-web-developer-docs/issues/360

Link to Google Doc: https://docs.google.com/document/d/17AI6MHRN6zer2mWWmnolfOS7TcX_Aq5Opqd084BmUqU/edit?tab=t.0#heading=h.l7ojtktjmdbx

### CODE REVIEW

File: Tutorial-Learn-How-to-Fix-a-CI-Flake-waitFor-Statements
Comment (U8NWXD) [inline Tutorial-Learn-How-to-Fix-a-CI-Flake-waitFor-Statements:1]: You're missing a `.md` at the end of this file. Please fix that so I can see the rendered Markdown

**Review by U8NWXD — CHANGES_REQUESTED** [review]

**Comment by U8NWXD** [comment]
Also, don't forget to add the tutorial to the sidebar in this PR

**Comment by Ash-2k3** [comment]
Hi @U8NWXD, I have addressed your comments, PTAL, Thanks!

File: Tutorial-Learn-How-to-Fix-a-CI-Flake-waitFor-Statements
Comment (Ash-2k3) [reply Tutorial-Learn-How-to-Fix-a-CI-Flake-waitFor-Statements:1]: Ah sorry, I have fixed it, PTAL!

**Review by U8NWXD — APPROVED** [review]

**Pull request merged**